src/aat/resources/features/F-063/F-063.feature

```diff
@@ -0,0 +1,45 @@
+@F-063
+Feature: F-063: Retrieve search input details for dynamic display
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-215
+  Scenario: should retrieve search inputs
```

Comment:
"should retrieve search inputs for Dynamic display"

Reply:
updated here and in Jira ticket 👍